### ISSUE TITLE
Got 404 when downloading the downgrade version of beatsaber , with loggin oculus.com

### ISSUE THREAD

**Issue by chrishzb** [ComputerElite/OculusDB#2]
hi , I am trying to download the older version of beat saber ,

but I found that it can not be done when I am using oculusdb.rui2015.me to download the apk

it seems all older version apk is missing  , I tried 1.21. 1.19.1 etc

I don't know how to continue the downgrade,
Very Looking Forwards for your help
many thanks

**Comment by chrishzb** [comment]
yes , I am make sure  I had log in 

https://securecdn.oculus.com/binaries/download/?id=7016048771802358

**Comment by chrishzb** [comment]
yes , I am make sure I had Log in on https://oculus.com/experiences/quest

**Comment by chrishzb** [comment]
yes , I am make sure I had log in on https://oculus.com/experiences/quest

and when I am require the apk on https://securecdn.oculus.com/binaries/download/?id=7016048771802358

I got 404 ,

`Failed to load resource: the server responded with a status of 404 ()`

Is just this game get wrong  or just my account get wrong ?

**Comment by ComputerElite** [comment]
You aren‘t loggen in them. Go on the Site I sent you and log in. If that doesn‘t work use Edge as YouTube Browser

**Comment by chrishzb** [comment]
<img width="848" alt="image" src="https://user-images.githubusercontent.com/2245508/172338505-3d5d0c2c-5115-41c2-b48e-27153bfbce07.png">
<img width="794" alt="image" src="https://user-images.githubusercontent.com/2245508/172338653-406f787e-b618-494d-9820-1518f081cc44.png">

Edge Browser , Make Sure Loggin on https://oculus.com/experiences/quest , 
Still got 404

**Comment by chrishzb** [comment]
Oh , you 're right , I got it
